src/Mono.Android/Android.Runtime/TypeManager.cs

```diff
@@ -13,9 +13,9 @@ public static string LookupTypeMapping (string[] mappings, string javaType)
 			return Java.Interop.TypeManager.LookupTypeMapping (mappings, javaType);
 		}
 
-		public static void RegisterType (string java_class, Type t)
+		public static void RegisterType (string java_class, Type t, bool ignoreExistingRegistration = false)
```

Comment:
Alternatively if we agree that JNIEnv should be updated to not call this but instead directly call the `Java.Interop.TypeManager` version then I can simply revert the change there and make that.